control/veda_deliver.py

```diff
@@ -101,7 +101,9 @@ def run(self):
         """
         if self.encode_profile == 'youtube':
             self._CLEANUP()
+            self.start_transcription(self._DETERMINE_STATUS())
             return None
+
         if self.encode_profile == 'review':
             return None
```

Comment:
what about this encoding?

Reply:
@yro can you share some detail about this?

Reply:
Everything gets an mp4 encoding, so leave this one off. We're trying to deprecate using youtube altogether, and a video would have already triggered transcripting via its mp4 encode, so this would be a duplicate.

Reply:
@muhammad-ammar This is the call I'm referring to -- since everything gets an mp4 encoding, this would be a duplicate call (right? I could be mistaken)

Reply:
@yro if you see the PR diff, we are not kicking transcription process in case of mp4 encode anymore. 

A little summary, we don't want any video encoding override Transcription statuses in Studio once the transcription has started. To achieve this, we have to wait for encodings to complete before we start transcription (Because if we do transcription on mp4 encoding then any other encoding that comes after mp4 encoding will overwrite video status from "transcript in progress" to something else which is undesirable and spread confusions among authors).

Do you think of any other approach that we could adopt instead?